systems/init/msgClient/internal/config.go

```diff
@@ -0,0 +1,31 @@
+package internal
+
+import (
+	"time"
+
+	uconf "github.com/ukama/ukama/systems/common/config"
+)
+
+type Config struct {
+	uconf.BaseConfig `mapstructure:",squash"`
+	DB               *uconf.Database   `default:"{}"`
+	Grpc             *uconf.Grpc       `default:"{}"`
+	Queue            *uconf.Queue      `default:"{}"`
+	Metrics          *uconf.Metrics    `default:"{}"`
+	Timeout          time.Duration     `default:"3s"`
+	HeathCheck       HeathCheckRoutine `default:"{}"`
+	System           string            `default:"init"`
```

Comment:
Can we make var `System` a mapstructure:"system"  in order to make the System variable configurable ?

Reply:
just define System in env that should be good enough